Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    name: Listen5
+    version: "1.2"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Listen5/36 CFNetwork/1406.0.4 Darwin/22.4.0'
+  os:
+    name: iOS
+    version: "16.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Listen5
+    version: "36"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'luminary.next/1.25 (Linux;Android 9) ExoPlayerLib/2.10.5'
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: mobile app
+    name: Luminary
+    version: "1.25"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Luminary/53.0 (Android 13; SM-S908U; samsung b0q; en)'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: Luminary
+    version: "53.0"
+  device:
+    type: smartphone
+    brand: Samsung
+    model: 'Galaxy S22 Ultra'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'LuminaryPreprod/27 CFNetwork/976 Darwin/18.2.0'
+  os:
+    name: iOS
+    version: "12.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Luminary
+    version: "27"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Luminary/325 CFNetwork/1404.0.5 Darwin/22.3.0'
+  os:
+    name: iOS
+    version: "16.3"
+    platform: ""
+  client:
+    type: mobile app
+    name: Luminary
+    version: "325"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Luminary/1.0'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Luminary
+    version: "1.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Megaphone.fm'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Megaphone
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Menucast/1.4 CFNetwork/720.1.1 Darwin/14.0.0 (x86_64)'
+  os:
+    name: Mac
+    version: "10.10"
+    platform: x64
+  client:
+    type: mobile app
+    name: Menucast
+    version: "1.4"
+  device:
+    type: desktop
+    brand: Apple
+    model: ''
+  os_family: Mac
+  browser_family: Unknown
+
+-
+  user_agent: 'Menucast/1.52 CFNetwork/1220.1 Darwin/20.3.0'
+  os:
+    name: iOS
+    version: "14.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Menucast
+    version: "1.52"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Messenger/168725151 CFNetwork/1107.1 Darwin/19.0.0'
+  os:
+    name: iOS
+    version: "13.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: Messenger
```

Comment:
Is this and the one bellow Facebook's Messenger? If yes, then this and the one below should be "Facebook Messenger" (there is already such an app in the detector). If not, then it's probably https://apps.apple.com/us/app/messengerx-app/id1530098687 and then the name should be "MessengerX"

Reply:
I couldn't find this app myself, but the lower number of downloads means it is probably MessengerX.